src/background/backup-old-wallet-salt.ts

```diff
@@ -0,0 +1,108 @@
+const walletSaltBackup = 'wallet-salt-backup';
+
+export function backupOldWalletSalt() {
```

Comment:
Sorry to nit naming, but I feel like using `Incorrect` rather than `Old` here and in the file name would make more sense?

Reply:
`backupIncorrectWalletSalt` is actually incorrect. It's the appPrivateKey derived from it that is incorrect, not the salt.
Note that this code is ephemeral, so it will be removed in maybe 1 month.

Reply:
O right, confused for a sec ...haha. 👍